charm/docs/tutorial/getting-started.md

```diff
@@ -2,7 +2,7 @@
 
 ## What you’ll do
 - Deploy the NetBox charm.
-- Use an external Redis server
+- Integrate with redis using redis-k8s charm.
```

Comment:
I'd say "Integrate with Redis using the redis-k8s charm." to be consistent with elsewhere, where we capitalise Redis.